web/docs/understand/operators/sources/export.md

```diff
@@ -0,0 +1,37 @@
+# export
+
+Retrieves events from a VAST node. The dual to [`import`](../sinks/import.md).
+
+## Synopsis
+
+```
+export
+```
+
+## Description
+
+The `export` operator retrieves events from a VAST node.
+
+:::note Flush to disk
+Pipelines starting with the `export` operator do not access events that are not
+written to disk.
+
+We plan to change this behavior in the near future. Until then, we recommend
```

Comment:
I don't think we do—it is absolutely intended that we only query passive partitions, so this second part of this note should be toned down a bit.

What we must add in the future, however, is the following:
- A mechanism to export continuously (i.e., retrieve all new partitions as they are registered at the catalog).
- Replace explicit flushing with an implicit flush in the import operator, and respect partition timeouts in the import operator.

Reply:
I removed the "We plan..." sentence in https://github.com/tenzir/vast/pull/3193/commits/45aaa7cf4310261cd85e18870d054ffae86b2149.

Should I write issues for those two points?